tools/importer-msgraph-metadata/pipeline/parser_models.go

```diff
@@ -294,6 +271,13 @@ func parseCommonModels(schemas openapi3.Schemas) (models Models, err error) {
 		}
 	}
 
+	// Several constants are presented as models, to clean these up we'll remove any models having no fields
```

Comment:
given they have no values, should we also be converting any references to string/etc fields?

Reply:
At the moment there don't appear to be any direct references to these counterfeit models, I've added additional checking to make sure (and also to ensure that model references in general do exist). These will fall back to an underlying type, additionally reverting to string where the type cannot be determined at all.